services/common/src/components/projectSummary/DocumentUpload.tsx

```diff
@@ -144,13 +129,13 @@ export const DocumentUpload: FC<DocumentUploadProps> = (props) => {
   };
 
   const fileUploadParams = {
-    mineGuid: initialValues.mine_guid,
-    projectGuid: initialValues.project_guid,
-    projectSummaryGuid: initialValues.project_summary_guid,
+    mineGuid: mine_guid,
+    projectGuid: project_guid,
+    projectSummaryGuid: project_summary_guid,
   };
 
   const tableDocuments =
-    initialValues?.documents?.map(
+    documents?.map(
```

Comment:
I think that having initialValues here was also making bugs (not showing files that weren't in initialValues)